pages/_document.tsx

```diff
@@ -9,6 +9,10 @@ export default class MyDocument extends Document {
             href="https://cdn.eds.equinor.com/font/equinor-font.css"
             rel="stylesheet"
           />
+          <link
+            href="https://cdn.eds.equinor.com/font/equinor-mono.css"
+            rel="stylesheet"
+          />
```

Comment:
Do you need this both in `styles/_variables.scss` and here?

Reply:
We are doing this for the Equinor font, but it looks like it isn't necessary